src/librustc_codegen_llvm/base.rs

```diff
@@ -75,6 +76,68 @@ pub fn write_compressed_metadata<'tcx>(
     }
 }
 
+pub fn write_idata_sections<'tcx>(
+    _tcx: TyCtxt<'tcx>,
+    raw_dylibs: &[RawDylibImports],
+    llvm_module: &mut ModuleLlvm,
+){
+    let (idata_llctx, idata_llmod) = (&*llvm_module.llcx, llvm_module.llmod());
+
+
+    let idata_7 = SmallCStr::new(".idata$7");
+    let idata_6 = SmallCStr::new(".idata$6");
+
+    for raw_dylib in raw_dylibs {
+        debug!("linking raw dylib - {:?}", raw_dylib);
```

Comment:
It's not actually doing any linking here. Maybe rather call it "creating raw dylib idata section"?